### PR TITLE
Improving proposals limits on summary and title

### PR DESCRIPTION
**Pull request by andreslucena**

### CODE REVIEW

File: db/migrate/20160217115259_change_limit_on_title_proposal.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class ChangeLimitOnTitleProposal < ActiveRecord::Migration
+  def up
+    change_column :proposals, :title, :string, :limit => 161
```
Comment (josepjaume) [inline db/migrate/20160217115259_change_limit_on_title_proposal.rb]: Why 161 instead of 160? xD

File: lib/proposal_xls_importer.rb [hunk 2]
```diff
@@ -13,7 +13,9 @@ def import
         if data[:category_name].present?  
           proposal_data = ProposalData.new(data)
           proposal_data.parse!
-          create_proposal(proposal_data.to_attributes)
+          proposal = create_proposal(proposal_data.to_attributes)
+          result = proposal.save
+          debugger unless result
```
Comment (josepjaume) [inline lib/proposal_xls_importer.rb]: Careful :)

File: lib/proposal_xls_importer.rb [hunk 3]
```diff
@@ -13,7 +13,9 @@ def import
         if data[:category_name].present?  
           proposal_data = ProposalData.new(data)
           proposal_data.parse!
-          create_proposal(proposal_data.to_attributes)
+          proposal = create_proposal(proposal_data.to_attributes)
+          result = proposal.save
```
Comment (josepjaume) [inline lib/proposal_xls_importer.rb]: Use `save!` here

**Comment by josepjaume** [comment]
Do you think we could improve the seeds so proposals have longer summaries? I can then work a bit on the CSS.

**Comment by andreslucena** [comment]
I've just added the seeds. I'm having a problem on my local machine with the title 
  title: Faker::Lorem.paragraph.truncate(150) 

I get an error on rake db:seed, that title it's too long (!!). Probably it's something that I'm missing.

**Comment by josepjaume** [comment]
@andreslucena I've found the bug :D. It turns out that `limit` magic inside of `Measurable` was actually caching class variables in the `Measurable` module instead of in the inherited class. That made `Debates` take presedence _in the seeds_ as it was called _first_. Tricky one.

Clever code is dumb code.

**Comment by andreslucena** [comment]
:+1:

**Pull request merged**